source/components-and-apis.rst

```diff
@@ -225,6 +229,16 @@ See more about how using APIs in :doc:`./using-api-interface`.
      - **/api/v1**
      - `API - x509-identity-mgmt`_
      - Yes
+   * - Kafka Websocket
```

Comment:
I think you can change  "Kafka Websocket" to "Kafka WS (Kafka Websocket)" or just "Kafka WS"